src/main/scala/thurloe/dataaccess/SendGridDAO.scala

```diff
@@ -21,8 +21,11 @@ trait SendGridDAO {
 
   def sendNotifications(notifications: List[Notification]): Future[List[Response]] = {
     Future.sequence(notifications.map { notification =>
-      lookupPreferredEmail(notification.userId) flatMap { preferredEmail =>
-        val email = createEmail(preferredEmail, notification.replyTo, notification.notificationId, notification.substitutions)
+      val toAddressFuture = notification.userId map (lookupPreferredEmail) getOrElse
+        Future.successful(notification.userEmail.getOrElse(throw new NotificationException("No recipient specified", Seq.empty, notification.notificationId)))
```

Comment:
seems like you should use the email sent in the request over the preferred email, this does the opposite